README.md

```diff
@@ -1 +1,114 @@
-# dcamp18
+# Drupalcamp Cebu 2018
+This is the source code for the official website of Drupalcamp Cebu 2018 based on Drupal 8.
+## Requirements
+* [Expresso PHP](https://github.com/expresso-php/expresso-php)
+* [Composer](https://getcomposer.org/download/)
+* [Docker](https://docs.docker.com/engine/installation/)
+* [Docker Compose](https://docs.docker.com/engine/installation/)
+## Local setup with Expresso PHP
+If it doesn't exists, create a folder docker in your home directory. This is where you will keep all your docker projects.
+Clone Expresso PHP in ~/docker
+```
+$ cd ~/docker
+$ git clone https://github.com/expresso-php/expresso-php.git drupalcampcebu2018
+```
+### Set Expresso PHP to match PROD: Nginx with PHP 7.0
```

Comment:
Not applicable. We are using PHP 7.2.

Reply:
kuya luc when I run php -v, it says that PHP 7.0.25. 
What do you mean we are using PHP 7.2 ?

Reply:
![phpversion](https://user-images.githubusercontent.com/36254805/38738260-456c212c-3f64-11e8-8a2f-faa4e30108e9.png)

Reply:
Hi @nhiieL , this is your laptop's PHP. If you /bin/bash into the containers and run the same command you should see PHP 7.2.